### ISSUE TITLE
Wagtail certificate expired

### ISSUE THREAD

**Issue by ekemeyer** [WGBH-MLA/ov-wag#181]
# Details 
Wagtail certificate expired. Firefox error message below:
```log
Websites prove their identity via certificates, which are valid for a set time period. The certificate for admin.ov.wgbh-mla.org expired on 6/11/2024.
 
Error code: SEC_ERROR_EXPIRED_CERTIFICATE
 ```

Submitted by: Peter Higgins
CC in communications: 
Priority: Urgent (within the next 1-2 days)
URL: 
Slack message thread:

**Comment by mrharpo** [comment]
Cert is fixed for now, but not sure if the renewal is fixed for good.

## Problem
After much trial and error, it looks like the `admin.ov.wgbh-mla.org` cert was not renewing because:
- The cert request failed the self-check with an error code of `400`
- It renewed fine as soon as I deleted the `ov-frontend-ingress`

My best guess so far is that the cert verification for `admin.ov.wgbh-mla.org` got eaten up by the frontend `ov.wgbh-mla.org`, so the renewal never got through to the validation key.

This might go away when we migrate to production and use an admin subdomain that is not a sub-subdomain of the primary site.

## TLDR
Pick a different top level subdomain for `ov-wag`, e.g: `ov-admin.wgbh-mla.org`
